src/lib_ccx/ccx_decoders_isdb.c

```diff
@@ -1345,12 +1340,8 @@ int isdb_parse_data_group(void *codec_ctx,const uint8_t *buf, struct cc_subtitle
 		isdb_log("ISDB group B\n");
 	}
 
-	isdb_log("ISDB (Data group) version %d\n",version);
-
 	buf++;
-	link_number = *buf++;
-	last_link_number = *buf++;
-	isdb_log("ISDB (Data group) link_number %d last_link_number %d\n", link_number, last_link_number);
+	*buf++;
```

Comment:
I see `*buf` incrementing twice..

Reply:
Should I increment it twice ? I think there is no need to increment in twice.